docs/MACOS_INSTALL.md

```
@@ -2,12 +2,61 @@
+### Option B: Native (no container)
+
+Install llama.cpp on the host and run with `--nocontainer`:
+
+```bash
+brew install llama.cpp
+ramalama --nocontainer run tinyllama
+```
+
+You can persist this in `$HOME/.config/ramalama/ramalama.conf`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)